components/mdns/Kconfig

```diff
@@ -10,13 +10,11 @@ menu "mDNS"
 
     config MDNS_MAX_SERVICES
         int "Max number of services"
-        range 1 64
         default 10
         help
             Services take up a certain amount of memory, and allowing fewer
             services to be open at the same time conserves memory. Specify
```

Comment:
```suggestion
            services to be open at the same time conserves memory. 
```

Reply:
I only removed the range of MDNS_MAX_SERVICES in this PR. This value still specifies the maximum amount of services.